### ISSUE TITLE
TODO : Zero stage 1,2 backward

### ISSUE THREAD

**Issue by hyeinhyun** [EleutherAI/oslo#130]
## Describe a TODO feature

- Change Zero logic about backward function
   as-is : optimizer.backward
   to-be : loss.backward
## Assignees

- hyein.hyun
